.circleci/config.yml

```diff
@@ -242,29 +270,46 @@ workflows:
   version: 2
   build_test_publish:
     jobs:
-      - lint-build-debug
-      - build-release
-      - build-release-x86
-      - build-osx
+      - build-base
+      - lint-build-debug:
+          requires:
+            - build-base
+      - build-release:
+          requires:
+            - build-base
+      - build-release-x86:
+          requires:
+            - build-base
+      - build-osx:
+          requires:
+            - build-base
       - build-release-binary-osx:
+          requires:
+            - build-base
           filters:
             tags:
               only: /.*/
             branches:
               ignore: /.*/
       - build-debug-binary-osx:
+          requires:
+            - build-base
```

Comment:
How does it work to use `build-base` which would be the linux image for osx builds?

Reply:
True it doesn't. Must have slipped in.